packages/kit/src/core/create_app/index.js

```diff
@@ -107,6 +107,28 @@ function generate_client_manifest(manifest_data, base) {
 	`);
 }
 
+/**
+ * Information about all page routes, in a language-agnostic JSON file.
+ *
+ * @param {ManifestData} manifest_data
+ * @returns {string}
+ */
+function generate_route_manifest(manifest_data) {
+	const page_routes = manifest_data.routes
+		.map((route) => {
+			if (route.type === 'page') {
+				const pattern = route.pattern.toString();
+				return {
+					pattern: pattern.slice(1, pattern.length - 1), // strip delimiters
+					params: route.params,
+					componentPath: route.a[route.a.length - 1]
```

Comment:
there won't be any `.svelte` files in the build directory, so is this really useful?

Reply:
True but it's an easy transformation to get from that to the HTML file. Basically replacing .svelte with .html and possibly adding an /index.